dss/util/security.py

```diff
@@ -99,20 +99,16 @@ def assert_authorized_issuer(token: typing.Mapping[str, typing.Any]) -> None:
     raise DSSForbiddenException()
 
 
-def assert_authorized_group(group: typing.List[str], token: dict) -> None:
-    if token.get(Config.get_OIDC_group_claim()) in group:
-        return
-    logger.info(f"User not in authorized group: {group}, {token}")
-    raise DSSForbiddenException()
-
-
-def assert_security(groups: typing.List[str]):
+def assert_security(*decorator_args, **decorator_kwargs):
     def real_decorator(func):
         @functools.wraps(func)
         def wrapper(*args, **kwargs):
-            assert_authorized_group(groups, request.token_info)
-            authz_handler = AuthHandler()
-            authz_handler.security_flow(authz_methods=['groups'], groups=groups, token=request.token_info)
+            decorator_kwargs.update(kwargs)
```

Comment:
Nit: I'm actually not sure how keyword arguments are shared inside the decorator. I would expect `kwargs` and `decorator_kwargs` to be the same. But I'm not sure. Either way, this line shouldn't be a problem.

Reply:
i was thinking the same thing, but it does out that the `decorator_kwargs` are the params that get passed in through the decorator ie `@my_decorator('wassup_dawg')`
while kwargs is what is getting passed into the function that is getting decorated in our case it would be something like `dss.api.bundles.get(uuid,version)` this is good though because for FLAC we can get what UUID the request was referring to :)

Reply:
OK that's awesome! Hard to keep proper context with everything but that all makes sense, thanks for investigating

Reply:
⚠️  this loses the `args` passed to the wrapper. most of the APIs have positional arguments, like for example file put:

https://github.com/DataBiosphere/data-store/blob/master/dss/api/files.py#L192